ydb/core/protos/flat_scheme_op.proto

```diff
@@ -1509,6 +1509,11 @@ message TApplyIf {
     optional uint64 PathVersion = 2;
     optional uint64 LockedTxId = 3;
     optional bool CheckEntityVersion = 4 [default = false];
+    optional ETypeOfPath CheckTypeOfPath = 5;
+
+    enum ETypeOfPath {
+        SubDomainOrExtSubDomain = 1;
+    }
```

Comment:
Can't see the reason to introduce new enum like that (and limited to a single path type option at that). What for?

There is already exist enum of all path types -- `EPathType`, in this exact .proto-file.